src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/resource/PrisonIncentiveLevelResource.kt

```diff
@@ -54,13 +55,21 @@ class PrisonIncentiveLevelResource(
     @Schema(description = "Prison id", example = "MDI", required = true, minLength = 3, maxLength = 6)
     @PathVariable
     prisonId: String,
+    @Schema(description = "Include inactive prison incentive levels", example = "true", required = false, defaultValue = "false", type = "boolean", pattern = "^[true|false]$")
+    @RequestParam(defaultValue = "false", value = "with-inactive", required = false)
+    withInactive: Boolean = false,
   ): List<PrisonIncentiveLevel> {
-    return prisonIncentiveLevelService.getActivePrisonIncentiveLevels(prisonId)
+    return if (withInactive) {
+      prisonIncentiveLevelService.getAllPrisonIncentiveLevels(prisonId)
```

Comment:
This would only include the levels for which there is a `(prisonId, levelCode)` row in the DB tho right?

There is no guarantee this list would be exhaustive and include all levels available/active across all the prison estate right?

Reply:
correct. but i didn't want to force-create levels in a prison using default values for something historic cos the defaults are unlikely to make sense… but to the consumer of the api would look like historical truth!

Reply:
this endpoint's documentation will now better explain the situation #398